go_broke/main.py

```diff
@@ -1,4 +1,5 @@
 import go_broke.aquisition as aq
+import data_formatter
```

Comment:
does this work?  do you need to do `.data_formatter` or `go_broke.data_formatter`?  i figured doing this wont find that file in the PYTHONPATH

Reply:
The IDE complained that it couldn't find data_formatter, but it did run just fine. It did the same thing with importing bollinger.

Reply:
If you use from data_formatter import * you can directly use the functions defined in the file.